libs/platform/src/lib/table/table.component.ts

```diff
@@ -1979,6 +1992,52 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
     }
 
     /** @hidden */
+    private _listenToFocus(): void {
+        if (!this.tableContainer || !this.focusableMock) {
+            return;
+        }
+
+        let shouldDelegateFocus = true;
+
+        this._subscriptions.add(
+            fromEvent(this.tableContainer.nativeElement, 'keydown').subscribe((event: KeyboardEvent) => {
+                if (this.loading) {
+                    event.preventDefault();
```

Comment:
this can potentially cause issues with table controls, that are outside the table body. Is it required to call `preventDefault`?

Reply:
Removed